www/ivaa07/sp2/public/js/quiz_code.js

```diff
@@ -0,0 +1,82 @@
+window.onload = function () {
+    let result = {};
+    let step = 0;
+    function showQuestion(questionNumber) {
+        document.querySelector('.question').innerHTML = quiz[step]['q'];
+        let answer = '';
+        for (let key in quiz[step]['a']) {
+            answer += `<li data-v='${key}' class="answer-letiant">${quiz[step]['a'][key]}</li>`;
+        }
+        document.querySelector('.answer').innerHTML = answer;
+
+    }
+
+    document.onclick = function (event) {
+        event.stopPropagation();
+        if (event.target.classList.contains('answer-letiant') && step < quiz.length) {
+            // event.target.data
+            if (result[event.target.dataset.v] != undefined) {
+                result[event.target.dataset.v]++;
+            }
+            else {
+                result[event.target.dataset.v] = 0;
+            }
+            step++;
+            if (step == quiz.length) {
+                document.querySelector('.question').remove();
+                document.querySelector('.answer').remove();
+                showResult();
+            }
+            else {
+                showQuestion();
+            }
+        }
+        if (event.target.classList.contains('reload-button')) {
+            location.reload();
+        }
+    }
+
+    function showResult() {
+        let key = Object.keys(result).reduce(function (a, b) { return result[a] > result[b] ? a : b });
+
+        let div = document.createElement('div');
+        div.classList.add('result');
+        div.id = "result_text";
+        div.innerHTML = answers[key]['description'];
+        document.querySelector('main').appendChild(div);
+
+        let img = document.createElement('img');
+        img.src = 'images/' + answers[key]['image'];
+        img.classList.add('result-img')
+        document.querySelector('main').appendChild(img);
+
+        let reloadButton = document.createElement('button');
+        reloadButton.innerHTML = 'Znovu';
+        reloadButton.classList.add('reload-button');
+        document.querySelector('main').appendChild(reloadButton);
+
+        let authorizeButton = document.createElement('button');

```

Comment:
tyhle tlacitka mi nefunguji

Reply:
toto je spíš chyba toho eso. Nahrala jsem několikrát nové verzi stránek ale stale se zobrazuje původní varianta. Mám to rozhraní u sebe na localhostu, do webu se to nepodařilo umístit, Google nedává povolení na ten Domain.